### PR TITLE
doc: Head unit updated for PUMP_CHART format section

### PR DESCRIPTION
**Pull request by equinor-schen**
## Why is this pull request needed?

This pull request is needed because of unclear definition of Head unit in PUMP_CHART

## What does this pull request change?

This pull request change the Head unit to M for the PUMP_CHART

### CODE REVIEW

File: docs/docs/about/modelling/setup/facility_inputs/pump_modelling/pump_charts.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ FACILITY_INPUTS:
     TYPE: PUMP_CHART_VARIABLE_SPEED
     UNITS:
         RATE: <rate unit, currently only AM3_PER_HOUR supported>
-        HEAD: <polytropic head unit, M, KJ_PER_KG, JOULE_PER_KG supported>
+        HEAD: <M supported>
```
Comment (olelod) [inline docs/docs/about/modelling/setup/facility_inputs/pump_modelling/pump_charts.md:79]:
```suggestion
        HEAD: <polytropic head unit, only M supported>
```

**Review by olelod — CHANGES_REQUESTED** [review]
I suggest we write it the same way as for the RATE

File: docs/docs/about/modelling/setup/facility_inputs/pump_modelling/pump_charts.md (hunk 1 above)
Comment (equinor-schen) [reply docs/docs/about/modelling/setup/facility_inputs/pump_modelling/pump_charts.md:79]: Thanks. It has been updated as Ole Peter suggested, 
HEAD: <polytropic head unit, only M supported>

Comment (olelod) [reply docs/docs/about/modelling/setup/facility_inputs/pump_modelling/pump_charts.md:79]: > Thanks. It has been updated as Ole Peter suggested, HEAD: <polytropic head unit, only M supported>

I think you have forgotten to push the update to here, just done it locally.

Comment (equinor-schen) [reply docs/docs/about/modelling/setup/facility_inputs/pump_modelling/pump_charts.md]: Thanks, Ole Peter. I had it updated again. Could you please to take a review if it was updated / pushed in the remote repository? The new updated is below.

**Review by olelod — APPROVED** [review]

**Pull request merged**